core-dev/packages/coq/coq.8.11.dev/opam

```diff
@@ -18,7 +18,7 @@ and homotopy type theory) and teaching.
 """
 
 depends: [
-  "ocaml" {>= "4.05.0"}
+  "ocaml" {>= "4.05.0" & < "4.11"}
```

Comment:
```suggestion
  "ocaml" {>= "4.05.0"}
```
I really don't see why this part has to be synced with the OCaml repo. The OCaml repo has fixed artifacts that can't be updated with support for new compilers (if the new compiler is incompatible), but the `v8.11` repo branch can be updated at any time.

Reply:
Fair.

What about the `-native-compiler` option, should I sync that to 8.12.dev and dev as well?

Reply:
I have no idea, see the long background to the OS X change in coq/coq#11178. Perhaps one of the release managers for 8.12 ( @Zimmi48 @ejgallego) could weigh in whether this will be fixed "properly" in the near future, or whether we should have this workaround in opam?